pypsi/completers.py

```diff
@@ -141,7 +141,7 @@ def path_completer(token, prefix=''):
     if not token:
         cwd = '.' + os.path.sep
         filename_prefix = ''
-    elif token[-1] == os.path.sep:
+    elif len(token) > 1 and token[-1] == os.path.sep:
         cwd = os.path.expanduser(token[:-1])
```

Comment:
I'm a little hesitant to change the if statement itself. How about something like this instead?

```suggestion
         cwd = os.path.expanduser(token[:-1] or os.path.sep)
```

That way the if still fires and we fall back to the root directory if `token == op.path.sep`.